### DISCUSSION TITLE
20231117-devteam-value/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#46] [Comments]
# 20231117-devteam-value/

안녕하세요 인프랩의 향로입니다. 최근 인프랩 팀은 적극적으로 개발자 채용을 하고 있습니다. 인프랩 채용 공고 지원하시는 분들 입장에서는 인프랩 개발팀은 어떤 것을 추구할까, 나와…

https://tech.inflab.com/20231117-devteam-value/

**Comment by eunsukko** [comment]
개발팀의 가치와 미션 나눠주셔서 감사합니다..!
어떤 개발자가 되고싶은지, 어떤 팀이 되면 좋을지 덕분에 돌아봅니다!

ps. 찬찬히 보다가 1. 개발팀의 미션 첫줄에 프로덕트 엔지니어가 프로적트로 오타인거 같아서 살짝 댓글남겨두어요~
팀이 중요하게 생각하는게 무엇인지 그려져서 내용 너무 좋았어요ㅎ!!

**Reply by jojoldu2inflab** [reply]
앗! 오타 제보 감사합니다 :) 
잘봐주셔서 감사합니다 :)

**Comment by JunilHwang** [comment]
"머리는 차갑게, 가슴은 뜨겁게" 라는 말이 생각나네요!! 👏👏
글을 쭉 읽다보니 향로님이 여태까지 블로그에 적었던 내용들이 머릿속에 주마등처럼(?) 스쳐갑니다 ㅎㅎ

**Comment by merong** [comment]
거를게 하나도 없는, 가슴과 머리 둘다 와 닿는 얘기들입니다. 좋은글 감사합니다.

**Comment by mystyle2006** [comment]
인프런 팀에서 일을 하면 어떨지 궁금하게 만드는 내용이네요 !! :)

**Comment by choirj91** [comment]
공감하는 글들이 많아 이렇게 처음 댓글을 남겨봅니다.

개발자로써, '개발'이라는 단어 대신 각자의 전문 분야의 단어를 대입하면 누구에게나 큰 도움이 되는 글들인 것 같아서 사내 전체 채널에도 해당 글 공유해봤습니닷. 🤩

가장 효율적이고, 좋은 성능의 개발. 이제는 가장 비효율적으로 보일 수 있는 코딩도 현재 당장의 상황에 따라서 가장 좋은 선택이 될 수 있다라는 것을 알게 된 시점부터는 역시 가장 중요하게 생각해야 하는 것이 에티튜드 인 것 같더라구요.

과거의 레거시 코드는 그 환경과 그 시기에서는 최고의 선택이었다...!! 🙃 

결국 프로는 전문성 너머의 스스로에 대한 태도와 컨디션 조절, 그리고 바라지 않고 내가 먼저 솔선수범하며, 나 자신을 존중하여 상대방에게 그 존중을 전염시키는 그 지점이지 않을까 라는 생각을 하게 됩니다.

좋은 사람을 찾지말고 좋은 사람이 되겠습니다.
감사합니다. 😍

**Comment by Sirius506775** [comment]
이 글을 읽은 오늘의 저는 너무 운이 좋은 사람같네요

개발자라는 카테고리를 떠나 나는 어떤 사람이었는지를 돌아보게 하고 방향성을 제시해주는 글인 것 같아요.

이렇게 성장할 수 있는 시간을 가질 수 있게 해주셔서 감사합니다. 향로님 :)

**Comment by Hoo93** [comment]
어떤 개발자를 원하는지 알려주시니
인프랩이 어떤 것을 지향하는지 알 수 있어 너무 좋았습니다.
인프랩을 목표로 열심히 준비 중입니다.
지금 회사의 맡은 프로젝트를 성실히 마무리하고 도전해 보겠습니다 !!

**Comment by Jake1152** [comment]
42seoul에서 설명회 해주신 것이 인상 깊어서 다시 채용공고 보러왔다가 
설명회에서 말씀해주신 좋은글 되새길 수 있네요
좋은글 고맙습니다 :)

**Comment by SemiProgramer** [comment]
짧은 책을 읽은 것 같은 느낌입니다.
짧은 글이지만 많은 경험을 느낄 수 있었습니다.

**Comment by young00000000** [comment]
지난 1년간 사람들과 함께 일 할때의 자세에 대한 복잠함이 조금 해결된것같아요
일찍 봤으면 조금 더 나은자세로 임했을것 같습니다 !

**Comment by oy-janghoesoo** [comment]
좋은 글 잘 봤습니다. 고맙습니다.

**Comment by kukjun** [comment]
잘 보고 갑니다! 👍

**Reply by moonpang** [reply]
많이 되돌아보고 반성하게 되네요.. 멋집니다.

**Comment by inhachoi** [comment]
좋은 글 잘 읽고 갑니다 :)
